specification/schemas/PredicateValue.json

```diff
@@ -0,0 +1,32 @@
+{
+  "type": "object",
+  "required": [
+    "value_type",
+    "value"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "value_type": {
+       "type": "string",
+       "enum": [
+         "path",
+         "scalar",
+         "predicate",
+         "resource"
+       ],
+       "example": "scalar"
+    },
+    "value": {
+      "type": "object",
```

Comment:
The value is not always an object, when the `value_type` is `scalar`, this can be a `number`, `string`, or `null`. And when the `value_type` is `path`, this is a `string`.